site/src/utils/slug.js

```diff
@@ -0,0 +1,78 @@
+import slugify from '@sindresorhus/slugify';
+import {SLUG_SEPARATOR} from '../../config';
+
+/* url-safe processor */
+
+export const urlsafeSlugProcessor = string =>
+	slugify(string, {
+		customReplacements: [	// runs before any other transformations
+			['$', 'DOLLAR'], // `$destroy` & co
+			['-', 'DASH'], // conflicts with `separator`
+		],
+		separator: SLUG_SEPARATOR,
+		decamelize: false,
+		lowercase: false
+	})
+		.replace(/DOLLAR/g, '$')
```

Comment:
nit: I might indent this line and the next one less

Reply:
I didn't actually touch that file at all, just copypasta from the Svelte site, so I'm going to leave it as-is :)